src/Parser/Deathknight/Unholy/Modules/Features/Apocalypse.js

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+
+import SPELLS from 'common/SPELLS';
+import SpellIcon from 'common/SpellIcon';
+import SpellLink from 'common/SpellLink';
+import { formatPercentage } from 'common/format';
+
+import Analyzer from 'Parser/Core/Analyzer';
+import Enemies from 'Parser/Core/Modules/Enemies';
```

Comment:
Currently you're not using this import, but there's a change I suggest below where you do. If you do, you want to be using `'Parser/Core/Modules/EnemyInstances'` instead of `'Parser/Core/Modules/Enemies'`. Those two modules are identical, except `EnemyInstances` is capable of discerning between different mobs with the same name. (Really, `Enemies` shouldn't exist at all, I plan at some point to remove it and leave only `EnemyInstances`)